conf/messages

```diff
@@ -373,7 +373,7 @@ ated.prev-period-summary.agent.no-previous-returns = You have no previous return
 ated.prev-period-summary.createRuturn.text = Create a new ATED return for the chargeable period
 
 ated.prev-period-summary.agent-appoint.text = You can appoint an agent to act on your behalf or create your own returns.
-ated.prev-period-summary.appoint-agent-info.text = If you want to appoint an agent you need to re-authorise your agent as the 64-8 and the ATED 1 does not cover the new online service.
+ated.prev-period-summary.appoint-agent-info.text = If you want to appoint an agent you need to re-authorise your agent as the 64-8 and the ATED 1 do not cover the new online service.
```

Comment:
Should we even still have this - it never gets used other than in the messages file as it is no longer rendered where user does not have any previous returns?

Reply:
Separate story to be raised to look at unused keys